### DISCUSSION TITLE
FrSky x10s express speaker hiss

### DISCUSSION THREAD

**Discussion by Richjanz** [EdgeTX/edgetx#4196] [Q&A]
After upgrading my x10s express to 2.9.0 I now have a hiss from the the speaker. the hiss is always there no matter where the valumes are set. Has anyone else had this problem? I upgraded to 2.9.1 hoping that would cure the problem, but it still exists.

**Comment by pfeerick** [comment]
Go into Radio Settings => Hardware and enable the "Mute if no sound" option if it's present (it should be an available option on the X10). 

![companion_yKCTArKyy3](https://github.com/EdgeTX/edgetx/assets/5500713/13153129-5d8f-4cb9-ba6a-4e4407b2b4bc)

**Reply by Richjanz** [reply]
I did as you suggested and that did the trick. Thanks
